js/ui/checkBox.js

```diff
@@ -52,7 +52,7 @@ CheckBoxContainer.prototype = {
         let min = minBoxWidth + minLabelWidth + this._spacing;
         let nat = natBoxWidth + natLabelWidth + this._spacing;
         [min, nat] = node.adjust_preferred_width(min, nat);
-        
+
```

Comment:
There's no issue with removing nearby line that is just whitespace.. we try to leave code cleaner than it was - so picking up the trash around the area you're working is ok.  It's the needless reformatting of swaths of code for no other reason than appearance that is frowned upon.